In [4]:
pip install pandas

  Using cached pandas-2.2.2-cp310-cp310-macosx_10_9_x86_64.whl.metadata (19 kB)
  Using cached numpy-2.0.0-cp310-cp310-macosx_14_0_x86_64.whl.metadata (60 kB)
  Using cached pytz-2024.1-py2.py3-none-any.whl.metadata (22 kB)
  Using cached tzdata-2024.1-py2.py3-none-any.whl.metadata (1.4 kB)
Using cached pandas-2.2.2-cp310-cp310-macosx_10_9_x86_64.whl (12.6 MB)
Using cached numpy-2.0.0-cp310-cp310-macosx_14_0_x86_64.whl (6.9 MB)
Using cached pytz-2024.1-py2.py3-none-any.whl (505 kB)
Using cached tzdata-2024.1-py2.py3-none-any.whl (345 kB)

[notice] A new release of pip is available: 24.1.1 -> 24.1.2
[notice] To update, run: pip install --upgrade pip
Note: you may need to restart the kernel to use updated packages.


In [5]:
import pandas as pd

In [13]:
df = pd.read_csv("tested.csv")
df


PassengerId  Survived  Pclass  \
0            892         0       3   
1            893         1       3   
2            894         0       2   
3            895         0       3   
4            896         1       3   
..           ...       ...     ...   
413         1305         0       3   
414         1306         1       1   
415         1307         0       3   
416         1308         0       3   
417         1309         0       3   

                                             Name     Sex   Age  SibSp  Parch  \
0                                Kelly, Mr. James    male  34.5      0      0   
1                Wilkes, Mrs. James (Ellen Needs)  female  47.0      1      0   
2                       Myles, Mr. Thomas Francis    male  62.0      0      0   
3                                Wirz, Mr. Albert    male  27.0      0      0   
4    Hirvonen, Mrs. Alexander (Helga E Lindqvist)  female  22.0      1      1   
..                                            ...     ...   ...    ...    ...   
413                            Spector, Mr. Woolf    male   NaN      0      0   
414                  Oliva y Ocana, Dona. Fermina  female  39.0      0      0   
415                  Saether, Mr. Simon Sivertsen    male  38.5      0      0   
416                           Ware, Mr. Frederick    male   NaN      0      0   
417                      Peter, Master. Michael J    male   NaN      1      1   

                 Ticket      Fare Cabin Embarked  
0                330911    7.8292   NaN        Q  
1                363272    7.0000   NaN        S  
2                240276    9.6875   NaN        Q  
3                315154    8.6625   NaN        S  
4               3101298   12.2875   NaN        S  
..                  ...       ...   ...      ...  
413           A.5. 3236    8.0500   NaN        S  
414            PC 17758  108.9000  C105        C  
415  SOTON/O.Q. 3101262    7.2500   NaN        S  
416              359309    8.0500   NaN        S  
417                2668   22.3583   NaN        C  

[418 rows x 12 columns]

In [ ]:
1. 6610402175 ฟาร์ฮาน่า มาเหล็ม
6610405930 เบญญภา ประคุณหังสิต
6610405972 พัทธนันท์ ขันตี

2. source : https://www.kaggle.com/datasets/brendan45774/test-file

3. แหล่งที่มา : human

4. รูปแบบการจัดเก๋็บ : ตาราง

5. จำนวนข้อมูลตัวอย่าง : 418 

6. สิ่งที่ต้องการทำนาย : โอกาสในการรอดชีวิต

7. สิ่งที่ใช้ในการทำนาย : survived, Pclass, sex, age, sibsp, parch, cabin

8. ต้องการเตรียมขอมูลส่วนไหนออกหรือไม่อย่างไร : ตัดข้อมูล NaN ออก

9. ข้อมูลที่ไม่น่าจะได้ใช้ในระบบต้นแบบ : Name, fare, Embarked, PassengerId

10. ข้อมูลควรนำไปเก็บในรูปแบบอื่นหรือไม่ : ไม่ควรเนื่องจากข้อมูลในรูปแบบตารางจะทำให้ง่ายต่อการอ่านข้อมูล

11. ประเด็นอื่นๆ ที่เกี่ยวข้องกับข้อมูล : -

12. การมีส่วนร่วมของสมาชิกในกลุ่ม : เยี่ยมจริงๆๆ

13. การเปิดเผยการใช้เครื่องมือปัญญาประดิษฐ์ : -